### ISSUE TITLE
Update navigation steps to add-ins deployment

### ISSUE THREAD

**Issue by itishnitsa** [OfficeDev/office-js-docs-pr#2484]
Settings > Services & add-ins has been removed awhile back. There is a shortuct to 'Integrated Apps', but add-in deployment has to be accessed by the special URL. Please update documentation. Deploying add-ins through 'Integrated Apps' is not ideal because some options, like make add-in available, are missing from that interface.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 41508035-2045-1706-9917-cd2da1e99cc7
* Version Independent ID: ea51ce32-18e2-68ed-ddc5-74f0d3d8a3d1
* Content: [Publish Office Add-ins using Centralized Deployment via the Microsoft 365 admin center - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/publish/centralized-deployment)
* Content Source: [docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/publish/centralized-deployment.md)
* Product: **non-product-specific**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for reporting this @itishnitsa.

@ElizabethSamuel-MSFT, can you please investigate?

**Comment by ElizabethSamuel-MSFT** [comment]
@exextoc Bhavin et al, can you provide guidance here?

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@itishnitsa We're investigating how best update this and hope to fix in the docs soon.

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@itishnitsa The updated deployment steps in the document should be published soon.

We're still investigating how best to address your comments re: missing options, etc.

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@itishnitsa We're still looking into your unaddressed comment and hope to have an update for you soon.

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@itishnitsa We have since removed this Centralized Deployment article. From our [Deploy and publish](https://docs.microsoft.com/office/dev/add-ins/publish/publish) overview article, we have included links to the Integrated Apps and Centralized Deployment content for the admin center. If you're still facing deployment issues, let us know by creating an issue from the bottom of the relevant page.

In the meantime, I'll go ahead and close this issue.

Thanks.
